common/src/main/java/com/microsoft/identity/common/internal/net/HttpResponse.java

```diff
@@ -28,11 +28,14 @@
 import java.util.List;
 import java.util.Map;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
 /**
  * Deprecated
  * <p>
  * Currently served as an adapter of {@link com.microsoft.identity.common.java.net.HttpResponse}
  */
+@SuppressFBWarnings("NM_SAME_SIMPLE_NAME_AS_SUPERCLASS")
```

Comment:
```suggestion
@SuppressFBWarnings("NM_SAME_SIMPLE_NAME_AS_SUPERCLASS",
         justification = "This class kept its original name to avoid breaking change during the refactoring process." +
                 "Once the process is done, this class will be removed entirely. ")
```

Reply:
already fixed in the commit above.